Resources/Server Info/Rules.txt

```diff
@@ -0,0 +1,17 @@
+[color=#a4885c]The[/color] [color=#ffd700]Golden[/color] [color=#a4885c]Rule.[/color] Admins can disregard any and all of these rules if they deem it in the best interest of the current round, server, and/or community at large. They will of course be held fully accountable for their actions if they exercise this privilege.
+
+[color=#a4885c]1.[/color] Don’t be a dick. We’re all here to have fun. Antagonists have leeway here since ruining the round is kind of their job, but try to keep it interesting if you’re antag.
+
+[color=#a4885c]2.[/color] Erotic Roleplay (ERP) is [color=#ff0000]not allowed under any circumstance[/color].
+
+[color=#a4885c]3.[/color] Harassment of other players for OOC/meta reasons is [color=#ff0000]not allowed[/color].
+	- Annoying somebody in a round for IC/RP reasons is fine. Doing it consistently across multiple rounds for OOC reasons (“I don’t like this 		  player” AKA metagrudging) to the point where it becomes seriously problematic for the other player(s) is [color=#ff0000]not[/color].
+
+[color=#a4885c]4.[/color] Real life hate speech/slurs are not allowed.
+	- You are [color=#ff0000]NOT allowed[/color] to be racist against skin colors or start shouting the n-word, or similar IRL problems.
+	- You [color=#00ff00]ARE allowed[/color] to be e.g. speciesist against in-game species (if we ever get any). Keep it interesting RP wise.
```

Comment:
probably not neceessary to include yet since we don't actually have any

Reply:
I think it's fine to include given it'll just need adding back in in 6-12 months anyway.